src/store/amp-store.js

```diff
@@ -17,7 +17,10 @@ Vue.use(Vuex);
 const state = {
     workflowSubmission:{
       loading:false,
```

Comment:
Are there any places in bundle creation we could use the "loading" icon?  Not sure how efficient creating/searching bundles is, but might not hurt to use the spinner just in case.